octobot_trading/personal_data/positions/states/liquidate_position_state.py

```diff
@@ -0,0 +1,49 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import octobot_trading.enums as enums
+import octobot_trading.personal_data.positions.position_state as position_state
+
+
+class LiquidatePositionState(position_state.PositionState):
```

Comment:
```suggestion
class LiquidatedPositionState(position_state.PositionState):
``` 
should we name it like this ?

Reply:
What do you suggest ?

Reply:
Right

Reply:
No because a position can have this state when its status is LIQUIDATING or LIQUIDATED